pages/common/composer.md

```diff
@@ -0,0 +1,31 @@
+# composer
+
+> Dependency Manager for PHP.
+
+- Interactively create a composer.json file:
```

Comment:
Not really sure how can make it more explicit. I checked "npm" tldr page which creates a package.json file and its also dont explain what a package.json is. composer.json and package.json works more or less the same.